src/components/NotFound.js

```diff
@@ -0,0 +1,10 @@
+import './NotFound.css';
+
+export default function NotFound() {
+  return (
+    <body className="notfound">
+      <h1>Ahh beans, looks like that is not a real color</h1>
```

Comment:
was hoping i'd get this if i hit '/RGB/500/500/500' but that'd be some tricky pain in the butt error handling